pkg/upgrade/config/v1alpha1/config.go

```diff
@@ -0,0 +1,199 @@
+/*
```

Comment:
Refer to earlier comments. Can we have below package:
`pkg/upgrade/v1alpha1/config.go`

Reply:
done